src/elfpy/simulators.py

```diff
@@ -186,9 +186,16 @@ def execute_trades(self, trades: list[tuple[int, list[MarketAction]]]) -> None:
         trades : list[tuple[int, list[MarketAction]]]
             A list of agent trades. These will be executed in order.
         """
+        number_of_trades = sum((len(agent_trades) for _, agent_trades in trades))
+        if number_of_trades == 0:
+            return
+        logging.info(
```

Comment:
I think this should be `logging.debug`. I know it's not consistent now, but I'd like the `info` logs to be daily or _maybe_ per block sumaries. Definitely not per trade.

Reply:
this is not per trade. it's right before the for loop that loops across trades. so leave it as info? i'm okay either way, but I thought it's a nice thing to have to delineate the passage of time in logs. it also only gets printed when there's a trade.

Reply:
left this out, since `log_market_step_string` is sufficient